src/codeu/chat/server/Model.java

```diff
@@ -23,11 +23,15 @@
 import codeu.chat.common.Time;
 import codeu.chat.common.User;
 import codeu.chat.common.Uuid;
+import codeu.chat.util.Logger;
```

Comment:
I added it to help with debugging, but a further explanation of what you mean would be helpful for me to learn!